t/integration/test_canvas.py

```diff
@@ -89,10 +89,10 @@ def test_group_chord_group_chain(self, manager):
 
         result = (before | connect | after).delay()
         result.get(timeout=TIMEOUT)
-        redis_messages = list(map(
+        redis_messages = [map(
```

Comment:
This change is invalid.
The purpose of the code is to turn a `map()` object into a list.
The code here is completely different. It changes the value of `redis_messages` into `[map()]` from `[b'<string converted to bytes if python2>', b'<another string converted to bytes if python2>']`.